internal/cloudapi/v2/handler.go

```diff
@@ -448,7 +453,11 @@ func (h *apiHandlers) getComposeStatusImpl(ctx echo.Context, id string) error {
 			ImageStatuses: &buildJobStatuses,
 			KojiStatus:    &KojiStatus{},
 		}
-		buildID := int(initResult.BuildID)
+
+		if initResult.BuildID < math.MaxInt {
```

Comment:
will this ever be false?

Reply:
My mistake.  That check should be `< 0`.